centreon/src/Core/TimePeriod/Infrastructure/API/FindTimePeriod/FindTimePeriodPresenter.php

```diff
@@ -23,35 +23,36 @@
 
 namespace Core\TimePeriod\Infrastructure\API\FindTimePeriod;
 
-use Core\Application\Common\UseCase\AbstractPresenter;
-use Core\Application\Common\UseCase\PresenterInterface;
-use Core\Infrastructure\Common\Presenter\PresenterFormatterInterface;
-use Core\TimePeriod\Application\UseCase\FindTimePeriod\FindTimePeriodResponse;
+use Centreon\Domain\Log\LoggerTrait;
+use Core\Application\Common\UseCase\ResponseInterface;
+use Symfony\Component\Serializer\Exception\ExceptionInterface;
+use Symfony\Component\Serializer\Serializer;
+use Symfony\Component\Serializer\SerializerInterface;
 
-class FindTimePeriodPresenter extends AbstractPresenter implements PresenterInterface
+class FindTimePeriodPresenter
```

Comment:
I no longer see any specific use case related concern in this presenter which means it can be the unique generic presenter and move into `Common`. Therefore this shouldn't prevent it to have an interface at least to keep the O/C principle and allow some decoration ... for logs for example if we want to have some.

For now I no longer see any usage of the LoggerTrait but I would probably be in favor of using it inside a decorator instead.